packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -147,37 +147,20 @@ export default (filters?: FileFilter[], onDismiss?: () => void) => {
                         dispatch(interaction.actions.downloadFiles());
                     },
                 },
-                {
-                    key: "move-files",
-                    text: "Move Files",
-                    title: "Move files between NAS Cache",
-                    disabled: !filters && fileSelection.count() === 0,
-                    iconProps: {
-                        iconName: "MoveToFolder",
-                    },
-                    subMenuProps: {
-                        items: [
-                            {
-                                key: "off-nas",
-                                text: "Off NAS Cache",
-                                title: "Move files off the NAS cache",
-                                onClick() {
-                                    // Placeholder for moving files off NAS Cache
-                                    console.log("Move files off NAS Cache");
-                                },
-                            },
-                            {
-                                key: "onto-nas",
-                                text: "Onto NAS Cache",
-                                title: "Move files onto the NAS cache",
-                                onClick() {
-                                    // Placeholder for moving files onto NAS Cache
-                                    console.log("Move files onto NAS Cache");
-                                },
-                            },
-                        ],
-                    },
-                },
+                ...(isQueryingAicsFms
```

Comment:
Lets move this option above "Download"

Reply:
resolved in [034549d](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/034549de1962d08622e9c4e1b8ea409741103aa5)